### ISSUE TITLE
Unable to navigate context menu with keyboard

### ISSUE THREAD

**Issue by naruaika** [naruaika/eruo-data-studio#6]
This is strange behavior because we can't navigate any context menus for the application with the keyboard. All context menus are affected, without exception e.g. whether the flag is set to NESTED or SLIDING.
